### PR TITLE
refactor: make sure add all runtime globals when process_runtime_requirements

### PR DESCRIPTION
**Pull request by ahabhgk**
## Summary

https://github.com/webpack/webpack/blob/87660921808566ef3b8796f8df61bd79fc026108/lib/Compilation.js#L3509-L3513

`set` can insert when iterating it

## Require Documentation?

- [ ] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

**Comment by ahabhgk** [comment]
!bench

**Comment by ahabhgk** [comment]
!eco-ci

**Comment by ahabhgk** [comment]
!diff

File: crates/rspack_plugin_runtime/src/runtime_module/css_loading.rs [hunk 1]
```diff
@@ -7,21 +7,22 @@ use rspack_identifier::Identifier;
...
+  get_chunk_runtime_requirements,
+  runtime_module::{render_condition_map, stringify_chunks},
+};
 
-#[derive(Debug, Default, Eq)]
+#[derive(Debug, Eq)]
 pub struct CssLoadingRuntimeModule {
   id: Identifier,
   chunk: Option<ChunkUkey>,
-  runtime_requirements: RuntimeGlobals,
```
Comment (ahabhgk) [inline crates/rspack_plugin_runtime/src/runtime_module/css_loading.rs:16]: `runtime_requirements` should use `get_chunk_runtime_requirements`, this is a copied value, but actually we need a reference, but using reference will break the borrow checker rules, so using `get_chunk_runtime_requirements` to get it

**Review by LingyuCoder — APPROVED** [review]

**Pull request merged**